subsys/net/conn_mgr/events_handler.c

```diff
@@ -24,7 +24,7 @@ static void conn_mgr_iface_events_handler(struct net_mgmt_event_callback *cb,
 {
 	int idx;
 
-	NET_DBG("Iface event %u received on iface %d (%p)", mgmt_event,
+	NET_DBG("Iface event %x received on iface %d (%p)", mgmt_event,
```

Comment:
+1 to this change, but please add `0x%x` prefix so that it is clear that the value is hex
Also we could optimize the flash usage a bit here as the strings are similar, so I suggest something like this
```
NET_DBG("%s event 0x%x received on iface %d (%p)", "Iface", mgmt_event,
```
and relevant changes in the two printouts below.

Reply:
done